main.go

```diff
@@ -63,6 +63,13 @@ var flags = []cli.Flag{
 		Usage:   "don't include the first H1 heading in Confluence output.",
 		EnvVars: []string{"MARK_H1_DROP"},
 	}),
+	altsrc.NewBoolFlag(&cli.BoolFlag{
+		Name:    "strip-linebreaks",
+		Value:   false,
```

Comment:
Should we enable it by default?

Reply:
It would be a change to the output, which is why I set it disabled by default. If it were me, I'd enable by default on the next major release, and remove the option on the release after that.

Reply:
Okay, makes sense, I'll switch it on once we release mark-10.x